index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div class="navbar">
+        <a href="#home" class="active">Home</a>
+        <a href="#contact">Contact</a>
+        <a href="#about">About</a>
+        <a href="javascript:void(0);" class="videos" onclick="myFunction()"></a>
+            <i class="showvideos"></i>
+    </div>
```

Comment:
I'd suggest using HTML5 semantic tags!
Instead of `<div>` try to use `<nav>`

Reply:
- [ ] (Optional) You did a great job, however kindly remember to always answer the "Why" are you requiring these changes.